tests_disabled.json

```diff
@@ -161,10 +161,6 @@
             "name" : "selftests.test_framework.ExampleTest.test_deproxy_srvclient",
             "reason" : "Disabled by issue #73"
         },
-        {
-            "name" : "http2_general",
-            "reason" : "Disabled by issue #261"
-        },
```

Comment:
We can't to enable whole `http2_general` suite, #261 has not been fixed yet.

Reply:
oh, sorry, my bad. I uncommented them to work with my test. I returned it back. For my test I created new directory.